api/src/test/java/org/pmiops/workbench/api/DisksControllerTest.java

```diff
@@ -206,22 +207,6 @@ public void testListPD() throws ApiException {
         .containsExactly(expectedGceDisk, expectedRStudioDisk);
   }
 
-  @Test
-  public void testListPD_nameNotMatchingPrefix() throws ApiException {
```

Comment:
why remove this test?

Reply:
This logic is nolonger needed because the current LeoService method is `listDiskByCreator`.

- Before https://github.com/all-of-us/workbench/pull/7221/files, this test is necessary because we use PD name prefix to figure out which user create that disk.
- After that previous PR, that test should have been renamed to `testListPD_audioInfoNotMatching`.
- With my current PR, this test is not needed at all.

Reply:
thanks